mm2src/coins/tendermint/tendermint_tx_history_v2.rs

```diff
@@ -750,11 +750,7 @@ where
             Ok(highest_height)
         }
 
-        let q = format!(
-            "coin_spent.spender = '{}' AND tx.height > {}",
-            self.address.clone(),
-            self.from_block_height
-        );
+        let q = format!("coin_spent.spender = '{}'", self.address.clone(),);
```

Comment:
If we fetched the full transaction history, changed the state to `WaitForHistoryUpdateTrigger` and then found out that there are transactions, we'll request the whole transaction history starting from the 1st page:
https://github.com/KomodoPlatform/atomicDEX-API/blob/d84d03a4aa971c83dcbf4dbe1811380e1ce5f98c/mm2src/coins/tendermint/tendermint_tx_history_v2.rs#L571-L578

What if we store the last fetched page in the context or between the states? 🤔

Reply:
It's a good suggestion, let me try to implement it in this PR 🙂